doc/interface/spatial_discretization_methods.rst

```diff
@@ -0,0 +1,127 @@
+Conversely, the numerical solution is more accurate with more discrete points.
+Thus, we trade computation time for approximation accuracy by specifying the parameters that determine the number of discrete points.
+For the FV scheme, the number of axial discrete points in the column is given by the number of volume cells ``NCOL``.
+For the DG scheme, the number of axial discrete points in the column is given by the number of polynomial interpolation nodes (= ``POLYDEG`` + 1) times the number of DG elements ``NELEM``.
+At each axial discrete point, the LRMP and GRM additionally employ a particle.
+These particles are also discretized, which increases the total number of DOF per axial discrete point, especially for the GRM where particles are fully resolved.
+The parameters for the GRM particle discretization are given for FV in ``NPAR`` and for DG in ``PAR_POLYDEG`` and ``PAR_NELEM``.
+
+Order of convergence
+--------------------
+
+The computational performance of a numerical method depends on its theoretical order of convergence.
+The order of convergence refers to the rate at which the method's approximation approaches the exact solution under refinement of the spatial grid.
+
+A higher-order method can be faster than a low-order method:
+Imagine a high- and a low-order method's approximation to exhibit similarly bad approximation accuracy due to a coarse spatial resolution.
+Refining the grid for both methods by the same number of discrete points improves the approximation accuracy of the higher-order method more than the other one.
+Thus, the low-order method requires more DOFs and ultimately more compute time to compute a solution of the same accuracy.
+
+The theoretical order of convergence is an asymptotic property, however.
+Having the exact solution, we can compute an experimental order of convergence (EOC) via the formula
+
+.. math::
+    :label: ModelColumn
+
+    \begin{aligned}
+        EOC_k = \frac{log(\varepsilon_{k+1} / \varepsilon_{k}) }{log(n_{k} / n_{k+1})},
+    \end{aligned}
+
+with :math:`\varepsilon_{k}` and :math:`n_{k}` denoting some error norm and the degrees of freedom of the kth approximation.
+The EOC approaches the theoretical order of convergence for :math:`k \rightarrow \infty` but is typically lower-for underresolved problems.
+High-order methods typically suffer from start-off problems and won't exhibit their high order until the grid is fine enough and a certain accuracy is already reached.
+That is, increasing the number of discrete points from, e.g., 2 to 4 typically does not improve the solution according to the theoretical order of convergence but by a much smaller EOC.
+The EOC is highly problem-dependent, and it is generally unknown when a high-order method will actually be faster than a lower-order method.
+Experience shows that higher-order methods work well for smooth solutions.
+
+The theoretical order of convergence for the CADET-FV scheme is fixed at 2.
```

Comment:
Give some information on why it is 2. Maybe also add some sources.

Reply:
I dont really want to go into to much detail but stay concise. A reference to our FV paper is provided